modin/experimental/core/execution/native/implementations/omnisci_on_native/test/test_dataframe.py

```diff
@@ -2215,5 +2220,23 @@ def test_str(self):
         assert str(mds) == str(pds)
 
 
+class TestCompare:
+    def test_compare_float(self):
+        kwargs = {
+            "align_axis": "columns",
+            "keep_shape": False,
+        }
+        data1 = random_state.randn(100, 10)
+        data2 = random_state.randn(100, 10)
+        columns = list("abcdefghij")
+        pdf1 = pandas.DataFrame(data1, columns=columns)
+        pdf2 = pandas.DataFrame(data2, columns=columns)
+        mdf1 = pd.DataFrame(data1, columns=columns)
+        mdf2 = pd.DataFrame(data2, columns=columns)
+        mcmp = mdf1.compare(mdf2, **kwargs)
+        pcmp = pdf1.compare(pdf2, **kwargs)
+        df_equals(mcmp, pcmp)
```

Comment:
The preferred way for testing is to use run_and_compare. E. g. it checks execution does not default to pandas.

Reply:
Fixed.